runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -122,3 +235,95 @@ TR::CompilationInfoPerThreadRemote::cacheResolvedMirrorMethodsPersistIPInfo(TR_R
 
    _resolvedMirrorMethodsPersistIPInfo->push_back(resolvedMethod);
    }
+
+/**
+ * Code to be executed on the JITServer to cache NULL value static class
```

Comment:
@brief Method executed by JITServer to remember NULL answers for `classOfStatic()` queries